### DISCUSSION TITLE
Issues with Amazon

### DISCUSSION THREAD

**Discussion by ItzPyrrhic** [Prince25/StockAlertBot#127] [Q&A] [answered]
I'm trying to monitor a single listing off Amazon but the request is being instantly blocked on the first attempt and is asking for captcha verification. I tried changing the intervals but that's not doing anything. I don't believe I'm doing anything wrong as the other sites I have tracked are working fine. Do I need to use a proxy for it to properly work with Amazon?

**Comment by Prince25** [comment, marked as answer]
Yes. Please take a look at this [comment](https://github.com/Prince25/StockAlertBot/issues/116#issuecomment-939393510) for more information.
